include/bluetooth/conn.h

```diff
@@ -169,6 +172,7 @@ enum {
  *  @param id Which local identity the connection was created with
  *  @param le LE Connection specific Info
  *  @param br BR/EDR Connection specific Info
+ *  @param rv LE version info
```

Comment:
This seems out of date. You're not adding any member called `rv` to the `bt_conn_info` struct.